components/Navbar/NavbarMobile/TabMobile.tsx

```diff
@@ -0,0 +1,65 @@
+'use client'
+
+import clsx from 'clsx'
+import Link from 'next/link'
+import { usePathname } from 'next/navigation'
+
+import CheckIcon from 'public/assets/icons/check.svg'
+import LockIcon from 'public/assets/icons/lock.svg'
+
+import { useLang, useStatus, useTranslations } from 'hooks'
+
+export default function Tab({
+  index,
+  count,
+  params,
+  challenge,
+  clicked,
+}: {
+  index: number
+  count: number
+  params: any
+  challenge: { lessonId: string; title: string }
+  clicked: any
+}) {
+  const { slug } = params
+
+  const lang = useLang()
+  const t = useTranslations(lang)
+  const pathName = usePathname()
+
+  const pathData = pathName.split('/')
+  const isRouteLesson = pathData.length === 4
+
+  const status = useStatus(slug, challenge.lessonId)
+
+  const challengeId = isRouteLesson ? pathData.pop().split('-')[0] : undefined
+  const isActive = challenge.lessonId === challengeId
+  const isLast = index == count - 1
+
+  return (
+    <Link
+      href={`/chapters/${slug}/${challenge.lessonId}`}
+      title={t(challenge.title)}
+      onClick={() => clicked()}
+      className={clsx(
+        'justify-left flex items-center border-t border-white/25 py-[9.5px] text-center text-[21px] text-white/50 transition duration-100 ease-in-out',
```

Comment:
Here a subpixel value (and arbitrary values) again, would be better if we can avoid those.

Reply:
Addressed in ea822132cc3d13887c40e033b9b5d1605f6ed579
Thanks for the suggestion!